pulp_ansible/tests/functional/api/collection/v3/test_sync.py

```diff
@@ -1,196 +1,139 @@
 """Tests related to sync ansible plugin collection content type."""
 
-from pulpcore.client.pulp_ansible import AnsibleRepositorySyncURL
+import pytest
 
-from pulpcore.tests.functional.utils import PulpTaskError
-
-from pulp_ansible.tests.functional.utils import (
-    gen_ansible_remote,
-    monitor_task,
-)
-from pulp_ansible.tests.functional.utils import TestCaseUsingBindings
 
 REQUIREMENTS_FILE = "collections:\n  - testing.k8s_demo_collection"
 
 
-class SyncCollectionsFromPulpServerTestCase(TestCaseUsingBindings):
-    """
-    Test whether one can sync collections from a Pulp server.
+@pytest.fixture(scope="class")
+def galaxy_remote(ansible_collection_remote_factory):
+    return ansible_collection_remote_factory(
+        url="https://galaxy.ansible.com",
+        requirements_file=REQUIREMENTS_FILE,
+        sync_dependencies=False,
+    )
+
 
-    This performs two sync's, the first uses the V2 API and galaxy.ansible.com. The second is from
-    Pulp using the V3 API and uses the content brought in from the first sync.
+@pytest.fixture(scope="class")
+def galaxy_repository(ansible_bindings, monitor_task, ansible_repository_factory, galaxy_remote):
+    repository = ansible_repository_factory(remote=galaxy_remote.pulp_href)
+    result = ansible_bindings.RepositoriesAnsibleApi.sync(repository.pulp_href, {})
+    monitor_task(result.task)
+    return repository
```

Comment:
The after-sync content-count check for the galaxy repository was dropped. (`assert len(first_content.results) >= 1`)
I think it could be done here in the fixture, or it was unimportant?

Reply:
I think it doesn't actually tell much.
We sync from that repository later and check that this "at least one item" get's there...